fvm/environment/event_emitter.go

```diff
@@ -29,11 +29,11 @@ type EventEmitter interface {
 var _ EventEmitter = NoEventEmitter{}
 
 // NoEventEmitter is usually used in the environment for script execution,
-// where no event should be emitted.
+// where emitting an event does nothing.
 type NoEventEmitter struct{}
 
 func (NoEventEmitter) EmitEvent(event cadence.Event) error {
-	return errors.NewOperationNotSupportedError("EmitEvent")
+	return nil
```

Comment:
is it useful if we log the event here?

Reply:
I was considering it, but there might be so many events happening, that it might completely fill the logs. Even just doing it in debug might be to much.